src/components/Issues/kite-service-check.ts

```diff
@@ -0,0 +1,6 @@
+const isKiteServiceAvailable=()=>{
+    //WIP
+    return true;
```

Comment:
call the Health API endpoint here from the API docs https://github.com/konflux-ci/kite/blob/main/packages/backend/docs/API.md.

Reply:
Please go through conditions doc https://github.com/konflux-ci/konflux-ui/blob/main/docs/conditions.md
you can check the `isKubearchiveEnabled` condition for reference to how it's done.
you need to register the condition in `register.ts`